R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1404,6 +1404,8 @@ test_that("column functions", {
   c20 <- to_timestamp(c) + to_timestamp(c, "yyyy") + to_date(c, "yyyy")
   c21 <- posexplode_outer(c) + explode_outer(c)
   c22 <- not(c)
+  c23 <- trunc(c) + trunc(c, "year") + trunc(c, "yyyy") + trunc(c, "yy") +
```

Comment:
this just makes sure it can be called and no error. for simple call to JVM we don't generally test the behavior/result